spec/03-plugins/23-aws-lambda/01-access_spec.lua

```diff
@@ -1,3 +1,4 @@
+local cjson = require "cjson.safe"
```

Comment:
Better not using the safe module here, and instead:
```lua
require "cjson"
```

Because using the safe module requires to do handle errors manually, which isn't done in the tests this is used in.